### PR TITLE
Dashboard hotfix

### LINKED ISSUE #119: Change DB_HOST to DB_IP
### Task

- [x] Change DB_HOST to DB_IP on dashboard

### User Story
As a developer, I want to have all the environment variables consistent across the whole project for maintainability.

### PR DESCRIPTION
**Pull request by fm1psy**
This PR closes issue #119 

hot fix caused by environment variables having different names to the AWS environment variables

### CODE REVIEW

**Review by FarihaChoudhury — DISMISSED** [review]
love it

File: terraform/README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ DB_USERNAME=your_db_username
 DB_PASSWORD=your_db_password
 
 DB_NAME=your_db_name
-DB_IP=your_db_host_ip_address
+DB_IP=your_DB_IP_ip_address
```
Comment (A-Faris) [inline terraform/README.md]:
```suggestion
DB_IP=your_db_host_ip_address
```

**Review by A-Faris — DISMISSED** [review]
Looks good

**Review by A-Faris — DISMISSED** [review]
All good

**Review by A-Faris — APPROVED** [review]
nice

**Pull request merged**